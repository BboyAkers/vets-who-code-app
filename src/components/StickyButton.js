import React from 'react'
import { Link } from 'gatsby'

const styles = {
  position: 'fixed',
  zIndex: 20000,
  display: 'flex',
  justifyContent: 'center',
  alignItems: 'center',
  boxShadow: '0 5px 40px rgba(0,0,0,0.16)',
  height: 60,
  width: 60,
  left: 20,
  bottom: 20,
  borderRadius: '50%',
  fontSize: 35
}

const StickyButton = () => (
  <Link to="/donate">
    <button className="btn btn-charity-default" style={styles}>
      <i className="fa fa-heart"></i>
    </button>
  </Link>
)

export default StickyButton
