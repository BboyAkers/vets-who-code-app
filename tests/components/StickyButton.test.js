import React from 'react'
import StickyButton from '../../src/components/StickyButton'

describe('<StickyButton />', () => {
  let wrapper

  beforeEach(() => wrapper = shallow(<StickyButton />))

  test('should render correctly', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
