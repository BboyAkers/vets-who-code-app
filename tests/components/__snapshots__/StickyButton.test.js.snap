// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<StickyButton /> should render correctly 1`] = `
<Link
  to="/donate"
>
  <button
    className="btn btn-charity-default"
    style={
      Object {
        "alignItems": "center",
        "borderRadius": "50%",
        "bottom": 20,
        "boxShadow": "0 5px 40px rgba(0,0,0,0.16)",
        "display": "flex",
        "fontSize": 35,
        "height": 60,
        "justifyContent": "center",
        "left": 20,
        "position": "fixed",
        "width": 60,
        "zIndex": 20000,
      }
    }
  >
    <i
      className="fa fa-heart"
    />
  </button>
</Link>
`;
